src/main/java/rcms/utilities/daqexpert/reasoning/logic/failures/HltOutputBandwidthExtreme.java

```diff
@@ -0,0 +1,63 @@
+package rcms.utilities.daqexpert.reasoning.logic.failures;
+
+import rcms.utilities.daqaggregator.data.DAQ;
+import rcms.utilities.daqexpert.ExpertException;
+import rcms.utilities.daqexpert.ExpertExceptionCode;
+import rcms.utilities.daqexpert.Setting;
+import rcms.utilities.daqexpert.reasoning.base.action.SimpleAction;
+import rcms.utilities.daqexpert.reasoning.logic.basic.Parameterizable;
+
+import java.util.Map;
+import java.util.Properties;
+
+public class HltOutputBandwidthExtreme extends KnownFailure implements Parameterizable {
+
+    /**
+     * upper end of range for expected  rate
+     */
+    private double max;
+
+    public HltOutputBandwidthExtreme() {
+        this.name = "Extreme HLT output bandwidth";
+        this.max = 0;
+
+        this.action = new SimpleAction("Are we running with the correct pre-scale column?",
+                "Talk to the trigger shifter and shift leader",
```

Comment:
You should not continue running in these conditions. Otherwise you risk problems with the NFS mounts on the FUs which can take a long time to recover. Talk to the trigger shifter and shift leader. Have them check the pre-scale column. Check the per-stream bandwidths in F3Mon. You may need to call the HLT DOC.

Reply:
No steps - just one single message.

Reply:
This message is too long for the current database scheme - single step is limited to 255 chars. I can increase this which will require to run alter statements on both production and development databases or we can simply split into suggestion of 2 steps:

1. You should not continue running in these conditions. Otherwise you risk problems with the NFS mounts on the FUs which can take a long time to recover. Talk to the trigger shifter and shift leader. Have them check the pre-scale column. 
2. Check the per-stream bandwidths in F3Mon. You may need to call the HLT DOC.

Please let me know if you have preference in this matter.